src/FtpExtractor.php

```diff
@@ -12,6 +12,9 @@
 use Psr\Log\LoggerInterface;
 use Symfony\Component\Filesystem\Filesystem;
 use Webmozart\Glob\Glob;
+use Retry\RetryProxy;
+use Retry\Policy\SimpleRetryPolicy;
+use Retry\BackOff\ExponentialBackOffPolicy;
 
 class FtpExtractor
```

Comment:
Chybí mi test - Mohlo by to být součást testu tady: https://github.com/keboola/ex-ftp/blob/master/tests/phpunit/ConnectionTest.php#L26

Reply:
Otestovat, že z logu padá ten retry - (viz např. https://github.com/keboola/db-extractor-common/pull/126#discussion_r321146496). 

Akorát ten test by se musel změnit z expectException an try/catch a ošetřit ten log tam...

Reply:
Jo tohle je moc dobrý nápad, udělal jsem to tak :)